src/react-apps/applications/runtime/src/features/form/layout/index.d.ts

```diff
@@ -4,8 +4,8 @@ export interface ILayoutEntry {
   hidden: boolean;
 }
 
-export interface ILayoutContainer extends ILayoutEntry {
-  children: [ILayoutContainer | ILayoutComponent];
+export interface ILayoutGroup extends ILayoutEntry {
+  children: [ILayoutGroup | ILayoutComponent];
 }
 
 export interface ILayoutComponent extends ILayoutEntry {
```

Comment:
Should the interface have itemType? (since it's deleted in formlayout.ts)

Reply:
Good catch, removed.